src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -182,7 +182,7 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
           title={STEPPER_STEP_LABELS.CONFIRM_PUBLISH}
           index={steps.indexOf(STEPPER_STEP_LABELS.CONFIRM_PUBLISH)}
         >
-          <HighlightStepperConfirmContent />
+          <HighlightStepperConfirmContent enterpriseId={enterpriseId} />
```

Comment:
nit: One of the reasons `enterpriseId` is still in the global Redux store is to avoid needing to prop drill `enterpriseId` down the component tree.

If a subcomponent needs `enterpriseId`, we should rely more on pulling it from the Redux store instead of passing it via props to sub-components (e.g., look at the `mapStateToProps` and `connect` usages)